x-pack/plugins/apm/public/components/alerting/rule_types/anomaly_rule_type/index.tsx

```diff
@@ -26,13 +28,18 @@ import {
   AnomalySeverity,
   SelectAnomalySeverity,
 } from './select_anomaly_severity';
+import { SelectAnomalyDetector } from './select_anomaly_detector';
+import { ANOMALY_DETECTOR_SELECTOR_OPTIONS } from '../../../../../common/rules/apm_rule_types';
+import { ApmMlDetectorType } from '../../../../../common/anomaly_detection/apm_ml_detectors';
+import { getApmMlDetectorLabel } from './get_apm_ml_detector_label';
 
 interface AlertParams {
   anomalySeverityType?:
     | ML_ANOMALY_SEVERITY.CRITICAL
     | ML_ANOMALY_SEVERITY.MAJOR
     | ML_ANOMALY_SEVERITY.MINOR
     | ML_ANOMALY_SEVERITY.WARNING;
+  anomalyDetectorTypes?: ApmMlDetectorType[];
```

Comment:
I wonder if we can align the names of type and variable (other things may have to be renamed as well if you do this).
```suggestion
  anomalyDetectorTypes?: AnomalyDetectorType[];
```